config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :users, controllers: { sessions: 'users/sessions' }
```

Comment:
Why we use this controllers in routes?

Reply:
We want some changing in login page of user. To use that only controller we can add it in routes.